server/src/main/java/keywhiz/service/daos/SecretSeriesMapper.java

```diff
@@ -36,9 +36,9 @@
   private final ObjectMapper mapper;
   private final GroupDAO groupDAO;
 
-  @Inject public SecretSeriesMapper(
+  public SecretSeriesMapper(
       ObjectMapper mapper,
-      @Readonly GroupDAO groupDAO) {
+      GroupDAO groupDAO) {
     this.mapper = mapper;
     this.groupDAO = groupDAO;
   }
```

Comment:
Rather than always using the read-only context, the injected GroupDAO re-uses the current DSLContext.